modules/graphdb/templates/00_wait_node_count.sh.tpl

```diff
@@ -11,6 +11,11 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+# Function to print messages with timestamps
+log_with_timestamp() {
```

Comment:
Try defining this once for all scripts, e.g. `00_helpers.sh` and then use `source ...` to load it in the beginning.

Reply:
done